OWML.ModHelper.Logging/Output.cs

```diff
@@ -0,0 +1,48 @@
+﻿using OWML.Common;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace OWML.ModHelper.Logging
+{
+    public static class Output
+    {
+        private static IModConsole _owmlOutput;
```

Comment:
`public static IModConsole OwmlOutput { get; private set; }`
Then you can remove methods SetOwmlOutput and GetOwmlOutput